### ISSUE TITLE
Twisted-aware kernel [question]

### ISSUE THREAD

**Issue by balta2ar** [ipython/ipython#9308]
Hello. How do I run twisted-friendly python2 kernel (the one that would allow to execute code that uses twisted)? I looks like in ancient versions of IPython twisted was supported, but I can't find any mentioning of it in any modern IPython/Jupyter version.

**Comment by Carreau** [comment]
The closest I can think of is https://github.com/takluyver/aiokernel . I think that @glyph (?) was interested and contact @rgbkrk and me to talk about that.

**Comment by glyph** [comment]
@Carreau Yeah, I'm definitely still interested.  Kinda busy with other stuff at the moment but I hope I can jump on it soon.
